packages/react-composites/src/composites/ChatComposite/ChatScreenPeoplePane.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ParticipantList, ParticipantMenuItemsCallback } from '@internal/react-components';
+import React from 'react';
+import { AvatarPersonaDataCallback } from '..';
+import { ParticipantContainer } from '../common/ParticipantContainer';
+import { useLocale } from '../localization';
+import { usePropsFor } from './hooks/usePropsFor';
+
+export type ChatScreenPeopelPaneProps = {
+  onFetchAvatarPersonaData: AvatarPersonaDataCallback | undefined;
+  onFetchParticipantMenuItems: ParticipantMenuItemsCallback | undefined;
+};
+
+export const ChatScreenPeoplePane = (props: ChatScreenPeopelPaneProps): JSX.Element => {
```

Comment:
This is a great example of how to write clean conditional compilation code - instead of adding /* conditional compilation*/ everywhere, we need a separated module then we don't have to write multiple conditional lines. (and that's the reason of why we only strip the following line of comment). Thanks for being a pilot and doing this exploration!  @dmceachernmsft 

In the meantime, I would like to involve @prprabhu-ms to discuss this code style a little bit
- Inline the conditional component in the same file or create separated file - I believe that depends on length of code, it looks good to have it in separated file cause it's 30 lines of code